### PR TITLE
Quarkus log format is fixed to match the format defined in logback.xm…

### PR DESCRIPTION
**Pull request by kathirsvn**
**What this PR does**:
Quarkus log format is fixed to match the format defined in logback.xml in the previous version of apis

**Checklist**
- [X] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [X] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: apis/sgv2-quarkus-common/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+quarkus:
+  log:
+    console:
+      format: "%-5p [%t] %d{yyyy-MM-dd HH:mm:ss,SSS} %F:%L - %s%e%n"
```
Comment (ivansenic) [inline apis/sgv2-quarkus-common/src/main/resources/application.yaml:4]: I don't think this will work? Ohh wow this does work..

Comment (ivansenic) [inline apis/sgv2-quarkus-common/src/main/resources/application.yaml:4]: Two things:

* `%s%e` could this simply be `%m` - `Renders the log message plus exception (if any).`
* does `%d{ISO8601}` not working?

**Review by ivansenic — APPROVED** [review]

File: apis/sgv2-quarkus-common/src/main/resources/application.yaml (hunk 1 above)
Comment (kathirsvn) [reply apis/sgv2-quarkus-common/src/main/resources/application.yaml:4]: Yes, tested all 3 apis (docs, rest, and graphql), the log format change is reflected.

Comment (kathirsvn) [reply apis/sgv2-quarkus-common/src/main/resources/application.yaml:4]: Sure, changing %s%e to %m now.
ISO8601 didn't work, tested that, so had to put the date format explicitly

**Pull request merged**